pywb/apps/frontendapp.py

```diff
@@ -431,14 +575,110 @@ def proxy_route_request(self, url, environ):
         """
         return self.proxy_prefix + url
 
+    def proxy_auto_fetch_worker(self, env, *args):
```

Comment:
This function is no longer needed now with the static prefix routing, right?

Reply:
done